python/paddle/fluid/tests/unittests/mkldnn/test_squeeze2_mkldnn_op.py

```diff
@@ -0,0 +1,162 @@
+# Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import unittest
+import numpy as np
+import paddle
+import paddle.fluid.core as core
+from paddle.fluid.tests.unittests.op_test import OpTest, OpTestTool, convert_float_to_uint16
+
+
+@OpTestTool.skip_if(core.is_compiled_with_cuda(),
+                    "CUDA has to be skipped because it forces dygraph")
+class TestSqueeze2OneDNNOp(OpTest):
+    def set_op_type(self):
+        self.op_type = "squeeze2"
+
+    def init_test_case(self):
+        self.ori_shape = (1, 3, 1, 40)
+        self.axes = (0, 2)
+        self.new_shape = (3, 40)
+
+    def set_inputs(self):
+        self.inputs = {"X": self.x}
+
+    def init_attrs(self):
+        self.attrs = {"axes": self.axes, 'use_mkldnn': True}
+
+    def set_outputs(self):
+        self.outputs = {
+            "Out": self.x.reshape(self.new_shape),
+            "XShape": np.random.random(self.ori_shape).astype("float32")
+        }
+
+    def setUp(self):
+        self.set_op_type()
+        self.init_test_case()
+        self.x = np.random.random(self.ori_shape).astype("float32")
+        self.set_inputs()
+        self.init_attrs()
+        self.set_outputs()
+
+    def test_check_output(self):
+        self.check_output_with_place(core.CPUPlace(), no_check_set=['XShape'])
```

Comment:
Dose setting `no_check_set=['XShape']` will not prevent checking for this output existence?

Reply:
XShape output is irrelevant in this op. Its only purpose is to transfer X tensor's dims between forward and backward pass